### ISSUE TITLE
Need to provide relative path to all evaluating files

### ISSUE THREAD

**Issue by qfox** [node-eval/node-eval#9]
```
$ mkdir qq
$ echo "require('../test.js');" > qq/yolo.js
$ echo "console.log('ok');" > test.js
$ node -p "require('node-eval')(require('fs').readFileSync('./qq/yolo.js', 'utf-8'), './qq/yolo.js')"
ok
module.js:327
    throw err;
    ^

Error: Cannot find module '../test.js'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at evalmachine.<anonymous>:1:1
    at Object.exports.runInThisContext (vm.js:54:17)
    at module.exports (/Users/qfox/--test/node_modules/node-eval/index.js:48:70)
    at [eval]:1:21
    at Object.exports.runInThisContext (vm.js:54:17)
    at Object.<anonymous> ([eval]-wrapper:6:22)
```

The first run ended with `ok`, but the second failed since it doesn't know where is the file placed.
Guess we should fix this.

**Comment by awinogradov** [comment]
+1

**Comment by qfox** [comment]
closed via #10
